.buildkite/scripts/build_and_upload_firecracker_packages.sh

```diff
@@ -61,10 +64,10 @@ for artifact_path in "${PACKAGES[@]}"; do
     artifact_name=$(basename "${artifact_path}")
     echo "--- :cloudsmith: Should we update '${artifact_name}' in '${UPSTREAM_REGISTRY}'?"
     version="$(get_version_from_artifact_metadata "${artifact_path}")"
-    tag="$(get_tag_from_artifact_metadata "${artifact_path}")"
+    input_sha256="$(get_input_sha256_from_artifact_metadata "${artifact_path}")"
 
-    echo "Checking if a package with tag ${tag} exists upstream..."
-    if ! present_upstream "${artifact_name}" "${tag}"; then
+    echo "Checking if a package generated with the same inputs exists upstream. SHA: ${input_sha256}"
+    if ! present_upstream "${artifact_name}" "${input_sha256}"; then
```

Comment:
If I'm reading correctly, I think the tag you add and search for is _just_ the checksum, and not a labeled checksum like `input-sha256:<CHECKSUM>`. If that's correct, I really think we need to include an indicator of what the tag is; otherwise when we're (say) browsing through the contents of the repository, we're just going to have a context-free string of randomness. It'll be even worse if at some point in the future we tag with some other kind of checksum.